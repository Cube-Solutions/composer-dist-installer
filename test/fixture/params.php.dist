<?php
return [
    'foo' => '{{Bar?|bar}}',
    'baz' => {{Number?|123}},
];