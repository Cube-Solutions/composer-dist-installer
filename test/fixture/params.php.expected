<?php
return [
    'foo' => 'bar',
    'baz' => 123,
];